mobile_verifier/src/reward_shares.rs

```diff
@@ -669,6 +672,7 @@ impl CoverageShares {
         self,
         reward_shares: DataTransferAndPocAllocatedRewardBuckets,
         epoch: &'_ Range<DateTime<Utc>>,
+        banned_radios: BannedRadios,
```

Comment:
It strikes me now that `OracleBoostingStatus` should probably be a part of `RadioInfo` the same way `SPBoostedRewardEligibility` is.

Reply:
That is better 73a0c61